pkgs/development/python-modules/flask-restful/default.nix

```diff
@@ -26,7 +26,7 @@ buildPythonPackage rec {
     hash = "sha256-zOxlC4NdSBkhOMhTKa4Dc15s7VjpstnCFG1shMBvpT4=";
   };
 
-  patches = lib.optionals (lib.versionAtLeast werkzeug.version "2.1.0") [
```

Comment:
this might be still worthwhile if people are overriding, not sure how long we should hold onto such logic though...

Reply:
Normally we don't consider such cases at all in ``pkgs/development/python-modules`` and remove workarounds as soon as an update lands. I also don't think we should do that because it increases maintenance.

Reply:
> I also don't think we should do that because it increases maintenance.

How would this increase maintenance?